codeSnippets/gradle.properties

```diff
@@ -1,45 +1,35 @@
 # sytleguide
-kotlin.code.style = official
-
+kotlin.code.style=official
 # mpp
 kotlin.native.binary.memoryModel=experimental
-
-
 # gradle configuration
-org.gradle.configureondemand = false
-
+org.gradle.configureondemand=false
 # versions
-kotlin_version = 1.9.10
-ktor_version = 2.3.5
-kotlinx_coroutines_version = 1.6.4
-kotlinx_serialization_version = 1.4.1
-kotlin_css_version = 1.0.0-pre.625
-exposed_version = 0.41.1
-h2_version = 2.1.214
-postgresql_version = 42.5.1
-hikaricp_version = 5.0.1
-ehcache_version = 3.10.8
-
-logback_version = 1.2.11
-slf4j_version = 2.0.9
-gce_logback_version = 0.127.15-alpha
-log4j_version = 2.19.0
-kotlinx_html_version = 0.8.1
-
-proguard_version = 7.3.0
-mockk_version = 1.13.2
-jte_version = 2.2.4
-junit_version = 4.13.2
-
-hamcrest_version = 2.2
-napier_version = 2.6.1
-guice_version = 5.1.0
-
-prometheus_version = 1.10.3
-
-dropwizard_version = 4.2.15
-
-bootstrap_version = 5.2.3
-swagger_codegen_version = 1.0.42
+kotlin_version=1.9.10
```

Comment:
It looks like autoformatting change, please consider reverting blank lines

Reply:
Yes, it was auto-formatted. I will update the Properties Code Style.